cmd/server/swagger.go

```diff
@@ -34,5 +34,6 @@ func setupSwaggerStaticConfig() {
 	docs.SwaggerInfo.Version = version.String()
 	docs.SwaggerInfo.Description = "Woodpecker is a simple CI engine with great extensibility.\n" +
 		"To get a personal access token (PAT) for authentication, please log in your Woodpecker server,\n" +
-		"and go to you personal profile page, by clicking the user icon at the top right."
+		"and go to you personal profile page, by clicking the user icon at the top right." +
+		"Swagger UI: https://ci.woodpecker-ci.org/swagger/index.html"
```

Comment:
Why do you want to show the url to the / another UI while you are currently in the UI?

Reply:
This was before #1984 and I guess it's better to do it there and remove it here 👍